BookReader/BookReader.js

```diff
@@ -2796,40 +2796,30 @@ BookReader.prototype.flipRightToLeft = function(newIndexL, newIndexR) {
 };
 
 BookReader.prototype.setMouseHandlers2UP = function() {
-    this.setClickHandler2UP( this.prefetchedImgs[this.twoPage.currentIndexL],
-        { self: this },
-        function(e) {
-            if (e.which == 3) {
-                // right click
-                if (e.data.self.protected) {
-                    return false;
+    var self = this;
+    ['L', 'R'].forEach(function(dir) {
```

Comment:
Curious, instead of binding these two individual functions separately like this, could you abstract the handler and pass in a direction with the event? ie:
```
var handler = function(e, direction) { 
  if (e.which == 3) {
    ...
  }
  ...
};

this.setClickHandler2UP(this.prefetchedImgs[self.twoPage['currentIndexL']]), { self: self, direction: 'L' }, handler);
this.setClickHandler2UP(this.prefetchedImgs[self.twoPage['currentIndexR']]), { self: self, direction: 'R' }, handler);
```

Reply:
I'm not sure if that is specifically the best solution, but I think we should avoid concatenating properties (e.g. `self.twoPage['currentIndex' + dir]`); that makes it a lot harder to grep to find where a given property is used.